packages/react-vapor/src/components/toast/Toast.tsx

```diff
@@ -67,6 +69,23 @@ export class Toast extends React.Component<IToastProps> {
         }
     }
 
+    private getInfoToken(type: string) {
+        let infoToken = '';
+        if (type === ToastType.Success || (_.isEmpty(this.props.className) && _.isEmpty(this.props.type))) {
```

Comment:
```suggestion
        if (type === ToastType.Success || (!this.props.className && !this.props.type)) {
```

Reply:
I guess this works, since the type of `className` and `type` is simply `string`, we don't need the `_.isEmpty`.